starter-code/index.html

```diff
@@ -3,40 +3,81 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <nav>
+      
+  <img class="spotifylogo" src="./images/spotify-logo.png" alt="spotifylogo">
+
+  <ul class="top-right">
+<li><a href="#">Premium</a></li>
+<li><a href="#">Discover</a> </li>
+<li><a href="#">Help</a> </li>
+<li><a href="#">Download</a></li>
+</ul>
+
+</nav>
+<div class="container">
+  <div class="containertitle">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.</p>
+<p>Listen to the right music, wherever you are.</p>
+</div>
+</div>
+
+<div class="container2">
+  <div class="container2head">
+<h2>What’s on Spotify?</h2>
+<hr>
+</div>
+</div>
+<div class="container3">
+<div class="MillofSong">
+<img src="./images/music-icon.png" alt="MillofSong">
+<h3>Millions of Songs</h3>
+<p>There are millions of</p>
+<p>songs on Spotify</p>
+</div>
+
+<div class="Hdmusic">
+<img src="./images/high-quality-icon.png" alt="Hdmusic">
+<h3>HD Music</h3>
+<p>Listen to music as if you</p>
+<p>were listening live</p>
+</div>
+
+<div class="StreamEverywere">
```

Comment:
Throughout this entire HTML page, you decided to name SOME of your class names using camel case? Please only use camel case in Javascript. The convention everyone follows for naming classes in HTML is to use all lower case and separate words with a hyphen.  So for example, your `"StreamEverywere"` should have been `"stream-everywere"` ... A few reasons why everyone follows this rule, is because CSS syntax is mostly case-insensitive. So to avoid errors others can have working on a team project with your code or reading your code, its best to just keep everything lower case and to separate words with a hyphen. It's best that you just realize that the best approach is the most compatible: use the same case in all code for a given CSS class, and don't create two or more CSS class names that differ only in case. It is also a good practice for readability. Also, even though there are other accepted ways to name classes in HTML, like using an underscore instead of a hyphen, for now lets stick with a hyphen.

Reply:
Also I think you misspelled this part of your class name:  "everywere"